File: Data Structures & Algorithms/single-number/submission-0.java
class Solution {
    public int singleNumber(int[] nums) {
        int answer = 0;
        if(nums.length == 1){
            return nums[0];
        }
        else{
            for(int i = 0 ; i < nums.length ; i++){
                answer = answer ^ nums[i];
            }
            return answer;
        }
    }
}

File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        for i in range(len(nums)):
            nums[i] = -nums[i]

        heapq.heapify(nums)
        answer = []

        for i in range(0,k):
            y = heapq.heappop(nums)
            answer.append(y)
            
        answer[k - 1] *= -1
        
        return answer[k - 1]